src/mailer/index.ts

```diff
@@ -3,6 +3,7 @@ import {
   signinMessage,
   resetMessage,
 } from './templates';
+const ride = process.env.VG_RIDE || 'viking.garage.app@gmail.com';
```

Comment:
@jonzee that env variable name is enigmatic, let's change it to something clear like `VG_BOOKING_EMAIL` or sth